cluster/gce/windows/README-GCE-Windows-kube-up.md

```diff
@@ -95,21 +98,24 @@ To teardown the cluster run:
 PROJECT=${CLOUDSDK_CORE_PROJECT} ./cluster/kube-down.sh
 ```
 
-#### 2b. Create a Kubernetes end-to-end (E2E) test cluster	
-If you have built your own release binaries following step 1, run the following	
-command:	
-```	
-PROJECT=${CLOUDSDK_CORE_PROJECT} ./hack/e2e-internal/e2e-up.sh	
-```	
+#### 2b. Create a Kubernetes end-to-end (E2E) test cluster
+If you have built your own release binaries following step 1, run the following
+command:
+```
+PROJECT=${CLOUDSDK_CORE_PROJECT} WINDOWS_NODE_OS_DISTRIBUTION=win2019 \
+  ./hack/e2e-internal/e2e-up.sh
+```
 
-If any e2e cluster exists already, this command will prompt you whether tears down and creates a new	one. To teardown existing e2e cluster only, run the command:
-```	
-PROJECT=${CLOUDSDK_CORE_PROJECT} ./hack/e2e-internal/e2e-down.sh	
-```	
+If any e2e cluster exists already, this command will prompt you whether tears
```

Comment:
s/whether tears down and creates/to tear down and create

Reply:
Fixed.